### PR TITLE
Add Identification Request Review Integration test

### PR DESCRIPTION
**Pull request by WaytoTrip**
Signed-off-by: sw-develop <psw00719@naver.com>

## Description
Add Identification Request Review Integration test

## Type of change
Please insert 'x' one of the type of change.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by astrod** [comment]
@soimkim This is a pull request about an intergration test related to Identification Request. Could you review it? I also check it.

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (astrod) [inline src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: Great

**Review by astrod — APPROVED** [review]
LGTM! After review from @soimkim , I am about to merge it. Thanks, @sw-develop

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (soimkim) [inline src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: Please add copyright text and license.
ex)
/*
 * Copyright (c) 2021 Soim Kim
 * SPDX-License-Identifier: AGPL-3.0-only 
 */

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (soimkim) [inline src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: How about setting the project name as "Test Project" and the project version as `UUID.randomUUID().toString()`?
I think it is necessary to filter the project created as a test by project name later and delete it.

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (soimkim) [inline src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: [Question] Will a project be created even if I enter an ID that is not registered as a user? 
"test2" is not a user added by default, so I wonder if a project will be created.

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (WaytoTrip) [reply src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: Should I write my name on copyright and the license in the same way as the example above?

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (WaytoTrip) [reply src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: The project is not stored in the database using @ Transactional.

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (WaytoTrip) [reply src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: Project is created regardless of UserId.

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (soimkim) [reply src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: @sw-develop , You should change the name to yours :-)

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (soimkim) [reply src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: Oh, I see. 👍

File: src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java
Comment (soimkim) [reply src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java]: Oh, I see.

**Comment by soimkim** [comment]
It's a minor thing, but I wish the commits were merged into one.
And add the copyright and license text as in the comment. 😃 

Except for these two things, it's LGTM.

**Comment by soimkim** [comment]
I have one more question. 
Considering that the status change test will be added to the 3rd party in the future, isn't it better to change the class name to RequestReviewIntegrationTest -> ProjectRequestReviewIntegrationTest?

**Comment by WaytoTrip** [comment]
Ok! I will change it.

**Comment by WaytoTrip** [comment]
@astrod @soimkim I modified the file. Please check.

**Comment by astrod** [comment]
The file is created by @sw-develop , So I guess that you might remain only your name. @soimkim, Can I merge it after fixing it?

**Review by soimkim — APPROVED** [review]

**Comment by soimkim** [comment]
Yep. 😄 
If it is a joint work between two people, it is better to include both copyrights. But I respect your opinion.

**Pull request merged**
